tests/__snapshots__/cli.test.ts.snap

```diff
@@ -1,8 +1,7 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`cli completion tests for cac > cli option completion tests > should complete option for partial input '{ partial: '--p', expected: '--port' }' 1`] = `
-"--port	Specify port
-:4
+":4
```

Comment:
This looks like we've lost something. These snapshots probably shouldn't have changed